integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/SlackResource.java

```diff
@@ -10,18 +13,21 @@
 
 public class SlackResource {
 
-    private final static String SLACK_URI = "https://slack.com/api/conversations.history?channel={channel}";
+    private final static String SLACK_URI = "https://slack.com/api/conversations.history";
     private final static String SLACK_CHANNEL = Utils.getSystemProperty("slack.channel");
     private final static String SLACK_TOKEN = Utils.getSystemProperty("slack.webhook.token");
     private final static String SLACK_WEBHOOK = Utils.getSystemProperty("slack.webhook.url");
 
     public static List<String> getListOfSlackMessages() {
+        final LocalDate yesterday = LocalDate.now(ZoneId.systemDefault()).minusDays(1);
         return given()
                 .auth()
                 .oauth2(SLACK_TOKEN)
                 .contentType(ContentType.JSON)
+                .queryParam("channel", SLACK_CHANNEL)
+                .queryParam("latest", yesterday.toEpochDay())
```

Comment:
Based on https://api.slack.com/methods/conversations.history it should be IMHO `oldest`

Reply:
Thinking whether one day isn't still quite a lot, what about 1 hour?